pynest/nest/lib/hl_api_types.py

```diff
@@ -647,11 +647,18 @@ def format_row_(s, t, sm, w, dly):
         MAX_SIZE_FULL_PRINT = 35  # 35 is arbitrarily chosen.
 
         params = self.get()
-        srcs = params['source']
-        trgt = params['target']
-        wght = params['weight']
-        dlay = params['delay']
-        s_model = params['synapse_model']
+        if len(params) == 0:
+            srcs = []
+            trgt = []
+            wght = []
+            dlay = []
+            s_model = []
+        else:
+            srcs = params['source']
```

Comment:
Maybe use `srcs = params['source'] if len(params) != 0 else []` to shorten? And the same for all the others. Now we have 2 `if` sentences to define `srcs` etc. (There is also an `if isinstance (srcs, int)` just below this.